### DISCUSSION TITLE
No images while saving pages on some websites

### DISCUSSION THREAD

**Discussion by bulleet** [wallabag/wallabag#6403] [wallabag.it]
Hi there!
Is there any solution for websites from which images are not saved?
Websites like:
- https://towardsdatascience.com/
- https://medium.com/
- https://www.nfx.com/
- https://apptractor.ru/
- https://teletype.in/
- https://app2top.ru/
- https://vc.ru/
- https://dtf.ru/

Thanks in advance!

**Comment by greengeek** [comment]
I have the same issue with any of the sites hosted on Medium.com, such as the top 2 in this list. All the text works, but the images are left blank. For an example, a random Medium.com article:

[https://medium.com/ctf-writeups/breaking-the-competition-bug-bounty-write-up-ca7cb7bc53f5](https://medium.com/ctf-writeups/breaking-the-competition-bug-bounty-write-up-ca7cb7bc53f5)

and a towardsdatascience.com which is also hosted on the Medium platform:

[(https://towardsdatascience.com/meta-ais-another-revolutionary-large-scale-model-dinov2-for-image-feature-extraction-1114b287eadd](https://towardsdatascience.com/meta-ais-another-revolutionary-large-scale-model-dinov2-for-image-feature-extraction-1114b287eadd)

The preview in the entries list shows an image, but no images are shown on the page itself. It appears they are just 1x1 gif place holders .

Happens both on wallabag.it and a manual install on Debian 12 using Wallabag 2.6.1. Accessing from Firefox and Chrome.

**Comment by j0k3r** [comment]
Conversation continue on the associated issue https://github.com/wallabag/wallabag/issues/6414
